wbooks/src/app/components/ButtonsApp/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { colorsGradient } from '@constants/colorsApp';
+
+import styles from './styles';
+
+export function BtnPrimary({ title, onPress }: any) {
```

Comment:
Nice!
Let's rename and separate this on individual folders in `@componentes/PrimaryButton` and `@componentes/SecondaryButton` with an `export default` each.